src/Button/DigitizeButton/DigitizeButton.tsx

```diff
@@ -12,15 +12,19 @@ import OlStyleStroke from 'ol/style/Stroke';
 import OlStyleFill from 'ol/style/Fill';
 import OlStyleCircle from 'ol/style/Circle';
 import OlStyleText from 'ol/style/Text';
-import OlInteraction from 'ol/interaction/Interaction';
 import OlInteractionDraw, { createBox } from 'ol/interaction/Draw';
 import OlInteractionSelect from 'ol/interaction/Select';
 import OlInteractionModify from 'ol/interaction/Modify';
 import OlInteractionTranslate from 'ol/interaction/Translate';
 import OlFeature from 'ol/Feature';
 import { never, singleClick } from 'ol/events/condition';
+import OlGeometry from 'ol/geom/Geometry';
+import { Select } from 'ol/interaction';
+import Draw from 'ol/interaction/Draw';
+import Modify from 'ol/interaction/Modify';
+import Translate from 'ol/interaction/Translate';
```

Comment:
Please rename the new imports like this. So we are in sync with the others.

```suggestion
import { Select as OlInteractionSelect } from 'ol/interaction';
import OlInteractionDraw from 'ol/interaction/Draw';
import OlInteractionModify from 'ol/interaction/Modify';
import OlInteractionTranslate from 'ol/interaction/Translate';
```

Maybe you can change the import of the event conditions to `import EventConditions from 'ol/events/condition'` too. So the execute code would look like `EventConditions.never`

Reply:
Done